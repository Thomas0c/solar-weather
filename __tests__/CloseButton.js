import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import CloseButton from '../lib/js/app/components/closeButton';

test('renders correctly', () => {
  const tree = renderer.create(<CloseButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
