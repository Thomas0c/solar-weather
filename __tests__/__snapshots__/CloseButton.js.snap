// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      Object {
        "alignSelf": "center",
        "bottom": -20,
        "position": "absolute",
      },
    ]
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityLiveRegion={undefined}
    accessibilityTraits={undefined}
    accessibilityViewIsModal={undefined}
    accessible={undefined}
    collapsable={undefined}
    hitSlop={undefined}
    importantForAccessibility={undefined}
    needsOffscreenAlphaCompositing={undefined}
    onAccessibilityTap={undefined}
    onLayout={undefined}
    onMagicTap={undefined}
    pointerEvents={undefined}
    removeClippedSubviews={undefined}
    renderToHardwareTextureAndroid={undefined}
    shouldRasterizeIOS={undefined}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#E77F6D",
        "borderRadius": 20,
        "borderWidth": 0,
        "height": 40,
        "justifyContent": "center",
        "overflow": "hidden",
        "width": 40,
      }
    }
    testID={undefined}
  >
    <Text
      accessible={true}
      adjustsFontSizeToFit={undefined}
      allowFontScaling={true}
      ellipsizeMode="tail"
      minimumFontScale={undefined}
      numberOfLines={undefined}
      onLayout={undefined}
      onLongPress={undefined}
      onPress={undefined}
      pressRetentionOffset={undefined}
      selectable={undefined}
      selectionColor={undefined}
      style={
        Object {
          "color": "#FFF",
          "fontFamily": "avenir",
          "fontSize": 16,
          "fontWeight": "700",
        }
      }
      suppressHighlighting={undefined}
      testID={undefined}
      textBreakStrategy={undefined}
    >
      X
    </Text>
  </View>
</View>
`;
